integration/oqs_openssl/README.md

```diff
@@ -1,7 +1,12 @@
 OQS-OpenSSL Integration Testing
 ===============================
 
-This directory contains a script for testing the OQS forks of OpenSSL with liboqs.
+This directory contains scripts for testing the OQS forks of OpenSSL with liboqs, using all supported algorithms. The README.md files for the OQS-OpenSSL forks ([1.0.2](https://github.com/open-quantum-safe/openssl/blob/OQS-OpenSSL_1_0_2-stable/README.md), [1.1.1](https://github.com/open-quantum-safe/openssl/blob/OQS-OpenSSL_1_1_1-stable/README.md)) describe the various key exchange and authentication mechanisms supported by each configuration.
+
+The script `run.sh` runs on Linux and macOS, and the PowerShell script `winrun.ps1` runs on Windows. First make sure you have **installed the dependencies** for the target OS as indicated in the [top-level testing README](https://github.com/open-quantum-safe/testing/blob/master/README.md).
```

Comment:
@christianpaquin The top-level testing README doesn't containing anything about dependencies for Windows.  Does anything need to be added there?  Or does it run without any special requirements?

Reply:
I guess so; I've created issue #13 to remember.